tools/depends/target/ffmpeg/FFMPEG-VERSION

```diff
@@ -1,5 +1,6 @@
 LIBNAME=ffmpeg
 BASE_URL=https://github.com/xbmc/FFmpeg/archive
-VERSION=2.8.6-Jarvis-16.1
+BRANCH=release/2.8-xbmc
```

Comment:
I tried it on my local rig and it worked - afaik last time there was just a -p missing in the mkdir command.